frb_rust/Cargo.toml

```diff
@@ -9,7 +9,7 @@ keywords = ["flutter", "dart", "ffi", "code-generation", "bindings"]
 categories = ["development-tools::ffi"]
 
 [dependencies]
-allo-isolate = "0.1.13-beta.5"
+allo-isolate = {version = "0.1.13-beta.5", git="https://github.com/lattice0/allo-isolate", branch = "anyhow", features=["anyhow"]}
```

Comment:
Oops we cannot have git dependencies for crate that is to be published

Upstream merged already?

Reply:
👀

Reply:
Yes!

Reply:
🎉